src/AccordionContainer/AccordionContainer.spec.tsx

```diff
@@ -122,6 +139,58 @@ describe('Accordion', () => {
         );
     });
 
+    it("can't remove an item when only one is expanded and the accordion doesn't allow zero expansions", () => {
+        const mock = jest.fn(() => null);
+        const itemFoo = { ...DEFAULT_ITEM, expanded: true, uuid: 'foo' };
+        const items = [itemFoo];
+
+        const instance = mount(
+            <Provider items={items}>
+                <Consumer>{mock}</Consumer>
+            </Provider>,
+        ).instance() as Provider;
+
+        expect(mock).toHaveBeenCalledWith(
+            expect.objectContaining({
+                items,
+            }),
+        );
+
+        instance.removeItem(itemFoo.uuid);
+
+        expect(mock).toHaveBeenCalledWith(
+            expect.objectContaining({
+                items: [itemFoo],
+            }),
+        );
+    });
+
+    it('can remove an item in an accordion that allows zero expansions when only one item is expanded', () => {
```

Comment:
I think this test is redundant. Item removal doesn't really have anything to do with this prop (nor `allowMultipleExpanded`, so that one can be removed too, I reckon).